<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(68, 68, 68);
color: rgb(250, 250, 250);
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="NoteGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>770</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::RubberBandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::NoAnchor</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Al Bayan</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>BPM</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="bpmBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="confilmBpmButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Confilm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="nowTime">
            <property name="font">
             <font>
              <family>Eurostile</family>
              <pointsize>29</pointsize>
              <italic>false</italic>
              <underline>false</underline>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>00000</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/new/prefix1/Play-icon.png</normaloff>:/new/prefix1/Play-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>ファイル</string>
    </property>
    <addaction name="actionSinkisakusei"/>
    <addaction name="actionOpen"/>
    <addaction name="actionJsonSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>編集</string>
    </property>
    <addaction name="moveaction"/>
    <addaction name="addaction"/>
    <addaction name="actionEraser"/>
    <addaction name="actionLN"/>
    <addaction name="environment_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSinkisakusei"/>
   <addaction name="actionOpen"/>
   <addaction name="actionJsonSave"/>
   <addaction name="moveaction"/>
   <addaction name="addaction"/>
   <addaction name="actionEraser"/>
   <addaction name="actionLN"/>
   <addaction name="actionPlayBySimulator"/>
   <addaction name="environment_action"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSinkisakusei">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/Document-icon.png</normaloff>:/new/prefix1/Document-icon.png</iconset>
   </property>
   <property name="text">
    <string>新規作成</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/Folder.png</normaloff>:/new/prefix1/Folder.png</iconset>
   </property>
   <property name="text">
    <string>譜面ファイルを開く</string>
   </property>
  </action>
  <action name="actionJsonSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/Save.png</normaloff>:/new/prefix1/Save.png</iconset>
   </property>
   <property name="text">
    <string>Json保存</string>
   </property>
  </action>
  <action name="addaction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/Pencil-icon.png</normaloff>:/new/prefix1/Pencil-icon.png</iconset>
   </property>
   <property name="text">
    <string>追加モード</string>
   </property>
   <property name="toolTip">
    <string>追加モード</string>
   </property>
  </action>
  <action name="moveaction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/hand-icon.png</normaloff>:/new/prefix1/hand-icon.png</iconset>
   </property>
   <property name="text">
    <string>移動モード</string>
   </property>
   <property name="toolTip">
    <string>移動モード</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/draw-eraser-icon.png</normaloff>:/new/prefix1/draw-eraser-icon.png</iconset>
   </property>
   <property name="text">
    <string>削除モード</string>
   </property>
  </action>
  <action name="actionLN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/LN_icon.png</normaloff>:/new/prefix1/LN_icon.png</iconset>
   </property>
   <property name="text">
    <string>ロングノーツモード</string>
   </property>
  </action>
  <action name="environment_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/Admin.png</normaloff>:/new/prefix1/Admin.png</iconset>
   </property>
   <property name="text">
    <string>環境設定</string>
   </property>
  </action>
  <action name="actionPlayBySimulator">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/Symbol_Play.png</normaloff>:/new/prefix1/Symbol_Play.png</iconset>
   </property>
   <property name="text">
    <string>playBySimulator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NoteGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>notegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playSound()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>907</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bpmBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeBpm(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>857</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>playSound()</slot>
  <slot>changeBpm(QString)</slot>
 </slots>
</ui>
